src/tests/JIT/Regression/JitBlue/Runtime_91175/Runtime_91175.cs

```diff
@@ -0,0 +1,275 @@
+        {
+            long long_154 = 84;
+            p_s4_s4_d1_f2_s4_d2_f3_143 = s_s4_s4_d1_f2_s4_d2_f3_75;
+            int __loopvar31 = s_loopInvariant - 10, __loopSecondaryVar31_0 = s_loopInvariant - 3;
+            for (;;)
+            {
+                if (__loopvar31 >= s_loopInvariant)
+                    break;
+                switch (((long)(long_88 & long_154)))
+                {
+                    case 84:
+                    {
+                        break;
+                    }
+                    case -1:
+                    {
+                        break;
+                    }
+                    case -2:
+                    {
+                        try
+                        {
+                        }
+                        finally
+                        {
+                            s_bool_22 = Avx.TestNotZAndNotC(((Vector256<int>)(((Vector256<int>)(v256_int_119 += ((Vector256<int>)(v256_int_119 - s_v256_int_60)))) - ((Vector256<int>)(s_v256_int_60 *= ((Vector256<int>)(s_v256_int_60 - Vector256<int>.Zero)))))), ((Vector256<int>)(((Vector256<int>)(Vector256.AsInt32(s_v256_ushort_59) - ((Vector256<int>)(v256_int_119 *= v256_int_119)))) - Avx2.ShiftRightArithmetic(Vector256<int>.AllBitsSet, Vector128<int>.AllBitsSet))));
+                        }
+                        break;
+                    }
+                    default:
+                    {
+                        break;
+                    }
+                }
+
+                // Added the following line so we don't loop infinitely
+                __loopvar31++;
+            }
+            return s1_128;
+        }
+    }
+
+    private void Method0()
+    {
+        unchecked
+        {
+            byte byte_229 = 0;
+            S1 s1_242 = new S1();
+            S2 s2_246 = new S2();
+            S4.S4_D1_F2.S4_D2_F3 s4_s4_d1_f2_s4_d2_f3_254 = new S4.S4_D1_F2.S4_D2_F3();
+            S4.S4_D1_F2.S4_D2_F3 s4_s4_d1_f2_s4_d2_f3_255 = s4_s4_d1_f2_s4_d2_f3_254;
+            S4 s4_257 = new S4();
+            S4 s4_258 = s4_257;
+            S5 s5_263 = new S5();
+            S5 s5_264 = s5_263;
+            s1_242 = Method1(s5_264, s2_246, ((byte)(byte_229 | ((byte)(((byte)(s_byte_23 + s_byte_23)) + ((byte)(byte_229 ^ s_byte_23)))))), out s4_s4_d1_f2_s4_d2_f3_134, ref s4_s4_d1_f2_s4_d2_f3_255, ref s_s3_73, s4_258);
+            return;
+        }
+    }
+
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        new TestClass().Method0();
+    }
+}
+/*
+
+Assert failure(PID 1312 [0x00000520], Thread: 31184 [0x79d0]): Assertion failed '!varTypeIsSIMD(vns->TypeOfVN(argVN))' in 'TestClass:Method1(TestClass+S5,TestClass+S2,ubyte,byref,byref,byref,TestClass+S4):TestClass+S1:this' during 'Do value numbering' (IL size 224; hash 0xf238d379; Tier1-OSR)
+    File: D:\git\runtime\src\coreclr\jit\valuenum.cpp Line: 6668
+    Image: D:\git\runtime\artifacts\tests\coreclr\windows.x64.Checked\tests\Core_Root\CoreRun.exe
+*/
```

Comment:
Use my test case from https://github.com/dotnet/runtime/issues/91175#issuecomment-1697408794 instead? I don't think we need a test this large for this.

Reply:
Done.